subprojects/docs/src/docs/release/release-features.txt

```diff
@@ -0,0 +1,4 @@
+🚀  JUnit 5 support
```

Comment:
Shouldn't this already be tested when running the build on the specific CI builds e.g. Windows? If it passes then it should fine.

Reply:
Well we want to make sure it also doesn't totally mess up, say, Ubuntu or Linux Mint. 

I would encourage use of emoji if possible because it significantly increases the likelihood that the message will be read, and gives a nice little visual summary (you may not even have to look at the words)

Reply:
I think it will be hard to test many different environments without setting them up on CI. Not sure if it's worth it at this point except if we wanted to test different environments manually for now. At least for Windows and Ubuntu we know that it works. Hoping for users to report back with issues if there are any.